harness/determined/constants.py

```diff
@@ -41,7 +36,9 @@
 # validation metrics.
 INTER_TRAIN_PROCESS_COMM_PORT_1 = int(str(os.getenv("INTER_TRAIN_PROCESS_COMM_PORT_1", "12360")))
 
-INTER_TRAIN_PROCESS_COMM_PORT_2 = INTER_TRAIN_PROCESS_COMM_PORT_1 + MAX_SLOTS_PER_AGENT
+INTER_TRAIN_PROCESS_COMM_PORT_2 = int(str(os.getenv("INTER_TRAIN_PROCESS_COMM_PORT_2", "12365")))
```

Comment:
default still `INTER_TRAIN_PROCESS_COMM_PORT_1` + `MAX_SLOTS_PER_AGENT`? could be useful for off cluster training eventually to have that default.